backends/kiibohd.py

```diff
@@ -214,6 +217,9 @@ def process( self, capabilities, macros, variables, gitRev, gitChanges ):
 		for result in range( 0, len( macros.resultsIndexSorted ) ):
 			self.fill_dict['ResultMacroList'] += "\tDefine_RM( {0} ),\n".format( result )
 		self.fill_dict['ResultMacroList'] += "};"
+		
+		results_count = len( macros.resultsIndexSorted );
+		print( "Results count: ", results_count )
```

Comment:
At least for the compiler, don't print anything unless the user needs to do/notice something.